### ISSUE TITLE
RFE: New resource for sslvserver_sslcertkey_binding

### ISSUE THREAD

**Issue by lhw** [citrix/terraform-provider-citrixadc#110]
**Describe the feature**
The one thing missing from the provider right now to make it easy to modularize is the option to add SNI certificates to an existing csvserver. 

Currently new SNI certificates have to be added to an existing csvserver resource
```hcl
resource "citrixadc_csvserver" "csv" {
  name        = "csv-dev"
  ipv46       = var.adc-vip
  servicetype = "SSL"
  port        = 443
  sslprofile  = "ssl_prof_tlsv12_tlsv13_sni"
  sslcertkey  = citrixadc_sslcertkey.default.certkey
  snisslcertkeys = [
    # every cert needs to be known here
  ]
}
```
Which means that if we create a publication type module for re-use we cannot use it with SSL certificates unless we want to create a new csverver for each publication.
Adding a sslvserver_sslcertkey_binding like this would be an easy option to have the modularization effect
```hcl
resource "citrixadc_sslvserver_sslcertkey_binding" "binding" {
  vservername = var.already-existing-csvserver
  snicert     = citrixadc_sslcertkey.sni.certkey
}
```
The only issue is that this would conflict with the parameter in the above csvserver. But I think a simple warning that both shouldn't be used at the same time would suffice.

I would generally just do the PR for this but if you don't want any code contributions this is the best I can do.

**Comment by lhw** [comment]
Do you see any option of adding this? Otherwise we would just have to maintain an in house fork. Which I'd rather like to avoid.

**Comment by lhw** [comment]
Alright. This looks another dead citrix project.

**Comment by vadharm** [comment]
Apologies for not coming back earlier. The project is active. Team will triage this and come back with fix soon. Will keep you posted.

**Comment by giorgos-nikolopoulos** [comment]
This is related to issues #81 #93.

Creating the `sslvserver_sslcertkey_binding` resource would solve this issue.

Will take it up when time permits.

**Comment by vadharm** [comment]
We have planned for the effort toward this. Will be targeting early to mid march for it.

**Comment by giorgos-nikolopoulos** [comment]
Fixed with #137
